carla_ros_bridge/src/carla_ros_bridge/ego_vehicle.py

```diff
@@ -127,6 +129,16 @@ def send_vehicle_msgs(self, frame, timestamp):
         vehicle_status.control.reverse = self.carla_actor.get_control().reverse
         vehicle_status.control.gear = self.carla_actor.get_control().gear
         vehicle_status.control.manual_gear_shift = self.carla_actor.get_control().manual_gear_shift
+        for wheel_idx in range(4):
+            wheel_location = VehicleWheelLocation(wheel_idx)
+            wheel_status = CarlaVehicleWheelStatus()
+            wheel_status.speed = self.carla_actor.get_wheel_speed(wheel_location) * 30.0 / math.pi
+            wheel_status.steer_angle = self.carla_actor.get_wheel_steer_angle(wheel_location)
+            wheel_status.pitch_angle = self.carla_actor.get_wheel_pitch_angle(wheel_location)
+            wheel_status.height = self.carla_actor.get_wheel_height(wheel_location)
+            wheel_status.lat_slip = self.carla_actor.get_wheel_lat_slip(wheel_location) * 180 / math.pi
```

Comment:
what are the units on these? This one has a rad to degree conversion but not the one below?

Reply:
accident left these conversions in when debugging. all angles are degrees as commented in the new msg definition.